tests/unit/AssociationType.spec.ts

```diff
@@ -0,0 +1,75 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import VueRouter from 'vue-router'
+import flushPromises from 'flush-promises'
+import sinon from 'sinon'
+import { getVuexStore } from '@/store/'
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import mockRouter from './MockRouter'
+import { AssociationType } from '@/components/common'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore()
+
+// Prevent the warning "[Vuetify] Unable to locate target [data-app]"
+document.body.setAttribute('data-app', 'true')
```

Comment:
Test without this. If not needed, please delete.

(I can never tell when it's needed or not.)

Reply:
Wasn't needed. Great!